inspektr/src/main/java/org/ccci/idm/user/inspektr/util/InspektrSerializationUtils.java

```diff
@@ -0,0 +1,34 @@
+package org.ccci.idm.user.inspektr.util;
+
+import com.google.common.base.Function;
+import com.google.common.base.Joiner;
+import com.google.common.base.MoreObjects;
+import com.google.common.collect.FluentIterable;
+import org.ccci.idm.user.Dn;
+import org.ccci.idm.user.Group;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class InspektrSerializationUtils {
+    private static final Joiner COMPONENT_JOINER = Joiner.on(",");
+    private static final Function<Dn.Component, String> COMPONENT_SERIALIZER = new Function<Dn.Component, String>() {
+        @Nullable
+        @Override
+        public String apply(@Nullable final Dn.Component input) {
+            return input != null ? input.type + "=" + input.value : null;
+        }
+    };
+
+    @Nonnull
+    public static String groupToString(@Nonnull final Group group) {
```

Comment:
Had you considered `groupToString ` adding to `UserAuditResourceResolver ` https://github.com/CruGlobal/idm-user-management/blob/master/inspektr/src/main/java/org/ccci/idm/user/inspektr/spi/UserAuditResourceResolver.java (or possibly adding and extending from `UserAndGroupAuditResourceResolver`)?